.changeset/cool-shrimps-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ai-sdk/provider-utils': major
```

Comment:
I used `major` since it's removing an API, but the pull request template calls explicitly for a `patch` type. Let me know what your preference is

Reply:
on v5 it does not matter, the release will be `major` regardless.

Reply:
Yes that's clear. But I think it might help with the release notes / upgrade guide when we flag breaking changes explicitly?